test/fakes/fake_sweyer_plugin_platform.dart

```diff
@@ -58,71 +57,80 @@ class FakeContentChannel implements ContentChannel {
   @override
   Future<void> insertSongsInPlaylist({
     required int index,
-    required List<Song> songs,
-    required Playlist playlist,
-  }) async {}
+    required List<int> songIds,
+    required int playlistId,
+  }) async {
+    final playlist = playlists!.firstWhere((playlist) => playlist.id == playlistId);
+    playlist.songIds.insertAll(index - 1, songIds);
```

Comment:
I was a bit surprised that the index is not zero based, but this seems by design:
https://github.com/nt4f04uNd/sweyer/blob/70711b3b2015fd4dce0fb5eea446ce0f01131b84/lib/routes/home_route/persistent_queue_route.dart#L244-L256

Are you sure this is correct? I could not find any more info about the [`PLAY_ORDER`](https://developer.android.com/reference/android/provider/MediaStore.Audio.Playlists.Members#PLAY_ORDER) entry.

Reply:
Yes, I'm sure. Both 0 and 1 give the same result.

Reply:
You can leave a comment about this here or the function signature

Reply:
Since a one based index is very unintuitive and we are going to move away from using the Mediastore for playlists anyways, I changed `insertSongsInPlaylist` to take a 0 based index and add one to it on the Android native side.